packages/client/hmi-client/src/components/code/tera-code.vue

```diff
@@ -15,36 +15,41 @@
 					/>
 				</section>
 				<section class="buttons">
-					<Dropdown
-						v-model="programmingLanguage"
-						:options="programmingLanguages"
-						@change="codeName = setFileExtension(codeName, programmingLanguage)"
-					/>
-					<FileUpload
-						name="demo[]"
-						:customUpload="true"
-						@uploader="onFileOpen"
-						mode="basic"
-						auto
-						chooseLabel="Load file"
-					/>
-					<Button label="Save" @click="saveCode()" />
-					<Button label="Save as new" @click="isCodeNamingModalVisible = true" />
-					<Button
-						label="Create model from code"
-						@click="isModelNamingModalVisible = true"
-						:loading="isCodeToModelLoading"
-					/>
-					<Button
-						label="Add dynamics"
-						:disabled="selectionRange === null"
-						@click="isDynamicsModalVisible = true"
-					/>
-					<Button
-						label="Remove all dynamics"
-						:disabled="!codeAsset?.files"
-						@click="removeAllDynamics"
-					/>
+					<template v-if="programmingLanguage !== ProgrammingLanguage.Zip">
+						<Dropdown
+							v-model="programmingLanguage"
+							:options="programmingLanguages"
+							@change="codeName = setFileExtension(codeName, programmingLanguage)"
+						/>
+						<FileUpload
+							name="demo[]"
+							:customUpload="true"
+							@uploader="onFileOpen"
+							mode="basic"
+							auto
+							chooseLabel="Load file"
+						/>
+						<Button label="Save" @click="saveCode()" />
+						<Button label="Save as new" @click="isCodeNamingModalVisible = true" />
+						<Button
+							label="Create model from code"
+							@click="isModelNamingModalVisible = true"
+							:loading="isCodeToModelLoading"
+						/>
+						<Button
+							label="Add dynamics"
+							:disabled="selectionRange === null"
+							@click="isDynamicsModalVisible = true"
+						/>
+						<Button
+							label="Remove all dynamics"
+							:disabled="!codeAsset?.files"
+							@click="removeAllDynamics"
+						/>
+					</template>
+					<template v-else>
+						<Button label="Download Zip" />
+					</template>
```

Comment:
Add a separate view for zip types for now...until we can get a new view for a file tree